runtimes/ruby-3.1/server.rb

```diff
@@ -76,7 +76,7 @@ def json(object)
     p e
     status 500
     content_type :json
-    return { code: 500, message: 'File not found or is not a valid ruby file.' }.to_json
+    return { code: 500, message: "Exception Occurred #{e.class}. Message: #{e.message}. Backtrace:  \n #{e.backtrace.join("\n")}" }.to_json
```

Comment:
```suggestion
    return { code: 500, message: "#{e.backtrace.join("\n")}" }.to_json
```
Just the trace is good enough 👍